canvas_modules/common-canvas/src/tooltip/tooltip.jsx

```diff
@@ -406,7 +411,14 @@ class ToolTip extends React.Component {
 		if (tooltipContent || link) {
 			tooltip = (
 				<Portal>
-					<div data-id={this.props.id} className={tipClass} aria-hidden={!this.state.isTooltipVisible} direction={this.props.direction}>
+					<div
+						role="tooltip"
+						id={`${this.uuid}-${this.props.id}`}
```

Comment:
Generating `UUID` in tooltip. Prefix it to tooltip id. Consuming component can pass a generic `id` in props. UUID makes sure tooltipId is unique for all.

Reply:
We need an `id` for `aria-labelledby` attribute to work. 
We can remove `data-id` now that `id` will be unique after using uuid. But many jest tests will break after removing data-id, so I kept both id and data-id.

Reply:
Ah, ok.